1주차/10828.py

```diff
@@ -0,0 +1,63 @@
+#10828
+"""
+push X: 정수 X를 스택에 넣는 연산이다.
+pop: 스택에서 가장 위에 있는 정수를 빼고, 그 수를 출력한다. 만약 스택에 들어있는 정수가 없는 경우에는 -1을 출력한다.
+size: 스택에 들어있는 정수의 개수를 출력한다.
+empty: 스택이 비어있으면 1, 아니면 0을 출력한다.
+top: 스택의 가장 위에 있는 정수를 출력한다. 만약 스택에 들어있는 정수가 없는 경우에는 -1을 출력한다. 
+
+"""
+
+def push(list,x):
+  list.append(x)
+
+def pop(list):
+  if len(list)!=0:
+    result=list[-1]
+    list.pop()
```

Comment:
```suggestion
    result=list.pop()
```
pop 함수는 마지막 원소를 제거하는 동시에 그 값을 반환해주니까 이렇게 한 줄로 쓸 수 있어요